data/campaigns/Eastern_Invasion/scenarios/01_Eastern_Invasion.cfg

```diff
@@ -137,7 +137,9 @@
         user_team_name= _ "Undead"
         {FLAG_VARIANT undead}
         type=Dark Sorcerer
-        id=Mal-Talar # this guy reappears in S06b, unless he dies here
+        #po: A male necromancer, reappears in S06b and can pursue the player to S07b.
+        #po: If killed in S01, a replacement (same unit but renamed as Mel Kevek) appears in S06b.
+        id=Mal-Talar
```

Comment:
Oops - this line needs to be above the po lines, so that the po lines are applied to the name= line.

Ah well, will do it tomorrow and add a changelog entry too.